src/llm-models/utils.ts

```diff
@@ -60,20 +66,24 @@ export function getErrorMessage(name: string, error: any): string {
     return error.error.error.message;
   } else if (name === 'ChromeAI') {
     return error.message;
+  } else if (name === 'OpenAI') {
+    return error.message;
   }
   return 'Unknown provider';
 }
 
 /*
  * Get an LLM completer from the name.
  */
-export function getSettings(name: string): JSONObject | null {
+export function getSettings(name: string): any {
```

Comment:
Maybe this `any` is fine for now, but we should maybe check if we can put `JSONObject | null` back as the return type.